zio-http/src/main/scala/zhttp/http/HttpContent.scala

```diff
@@ -10,4 +11,20 @@ sealed trait HttpContent[-R, +A] extends Product with Serializable
 object HttpContent {
   final case class Complete[A](data: A)                        extends HttpContent[Any, A]
   final case class Chunked[R, A](data: ZStream[R, Nothing, A]) extends HttpContent[R, A]
+
+  sealed trait FormDataContent extends Product with Serializable
+
+  // TODO: Make the environment R instead of hardcoded Blocking.
+  // It's set to Blocking for now because we need it when reading from a tmp file.
+  final case class FileData(name: String, contentType: String, content: ZStream[Blocking, Throwable, Byte])
+      extends FormDataContent
```

Comment:
```suggestion
  final case class FileData[R](name: String, contentType: String, content: ZStream[Blocking, Nothing, Chunk[A]])
      extends FormDataContent
```

Why should the stream throw?